js/src/util/validation.js

```diff
@@ -42,9 +44,14 @@ export function validateAbi (abi) {
   try {
     abiParsed = JSON.parse(abi);
 
-    if (!util.isArray(abiParsed)) {
+    if (!apiutil.isArray(abiParsed)) {
```

Comment:
Honestly see no issue with the age-old `Object.prototype.toString.call(obj) === '[object Array]` underlying implementation there. Could also use lodash isArray here, etc. Don't see benefit of the one over the other, if the underlying implementation is an issue, would then see that adjusted since it is used is multiple places.